bin/buildwatch.js

```diff
@@ -2,83 +2,73 @@ const sane = require('sane');
+const { CONTAINER_NAME } = process.env;
 
 const getContainerId = async () => {
-  if(!CONTAINER_NAME) {
-    console.log('Please set CONTAINER_NAME environment variable:\n CONTAINER_NAME=your_name_here npm run postbuild:lib:watch');
+  if (!CONTAINER_NAME) {
+    console.log(
+      'Please set CONTAINER_NAME environment variable:\n CONTAINER_NAME=your_name_here npm run postbuild:lib:watch'
+    );
     return false;
   }
   return await exec(`docker ps --quiet --filter "name=${CONTAINER_NAME}"`)
-    .then(({stdout, stderr}) => {
-      if(stderr) {
+    .then(({ stdout, stderr }) => {
+      if (stderr) {
         console.log(stderr);
         return false;
       }
-      const containerIds = stdout.split('\n').filter(el => el.length > 0)
-      if(containerIds.length === 1) {
-      } else if(containerIds.length < 1) {
+      const containerIds = stdout.split('\n').filter(el => el.length > 0);
+      if (containerIds.length === 1) {
+      } else if (containerIds.length < 1) {
         console.log(`No container with a name like ${CONTAINER_NAME} running`);
         return false;
-      } else if(containerIds.length > 1) {
-        console.log(`Multiple containers with names like ${CONTAINER_NAME} running: ${containerIds.reduce((acc, cur) => acc + cur + ',', '')}`);
+      } else if (containerIds.length > 1) {
+        console.log(
+          `Multiple containers with names like ${CONTAINER_NAME} running: ${containerIds.reduce(
+            (acc, cur) => acc + cur + ',',
+            ''
+          )}`
+        );
         return false;
       }
       return containerIds[0];
     })
-    .catch((reason) => console.log(reason));
-}
+    .catch(reason => console.log(reason));
+};
 const buildWatcher = sane('dist/ngx-prx-styleguide');
 
 buildWatcher.on('change', async (filepath, root, stat) => {
   // The final file updated by the Angular build is the package.json
-  if(filepath === 'package.json') {
+  if (filepath === 'package.json') {
     console.log(`${filepath} updated, Angular build complete`);
     console.log(`Building assets`);
     const assetBuild = await exec('npm run build:lib:assets')
-      .then(({stdout, stderr}) => {
-        if(stderr) {
+      .then(({ stdout, stderr }) => {
+        if (stderr) {
           console.log(stderr);
           return false;
         }
         console.log(stdout);
         return true;
       })
-      .catch((reason) => console.log(reason));
+      .catch(reason => console.log(reason));
 
     const containerId = await getContainerId();
 
-    if(containerId) {
+    if (containerId) {
       console.log(`Found running ${CONTAINER_NAME} container: ${containerId}`);
-      exec(`docker cp dist/ngx-prx-styleguide/. ${containerId}:/app/node_modules/ngx-prx-styleguide`)
-        .then(({stdout,stderr}) => {
-          if(stderr) {
-            console.log(stderr);
-          }
-          console.log('copied successfully');
-          console.log(stdout);
-        });
```

Comment:
Sorry about all the formatting changes and not making a separate commit. Change is here: It didn't work to just copy it over anymore, bunch of errors. So now it uses `npm pack` to tar up the build in the dist directory then it does a copy and exec's an install on the docker container.